src/qt/bitcoin.cpp

```diff
@@ -348,7 +347,7 @@ BitcoinApplication::~BitcoinApplication()
 #ifdef ENABLE_WALLET
 void BitcoinApplication::createPaymentServer()
 {
-    paymentServer = new PaymentServer(this, true, GetBoolArg("-enable-bip70", false));
+    paymentServer = new PaymentServer(this, true);
```

Comment:
Not for now, but we'll want to remember we once supported this flag when we move to 1.21 or reimplement/cherry-pick the code that checks for unsupported CLI arguments.

Reply:
Added your note to #1313